docs/lib/auth/fragments/flutter/signin_web_ui/10_cli_setup.md

```diff
@@ -0,0 +1,24 @@
+In terminal, navigate to your project, run `amplify add auth` (or if you've already added it, run `amplify update auth`), and choose the following options:
```

Comment:
```suggestion
In the terminal, navigate to your project and run `amplify add auth` (or `amplify update auth`, if you've already added it.) Choose the following options:
```

Reply:
This was taken directly from android docs.